src/locales/en.json

```diff
@@ -224,6 +224,8 @@
     "api-insufficient-rights": "Your user does not have sufficient rights to perform this action. Please contact your administrator.",
     "api-invalid-credentials": "The supplied login credentials were incorrect.",
     "api-auth-error": "An error occurred while logging you in: {cause}",
+    "api-auth-transient-error": "A (temporary) error occured while trying to authenticate your request. Please try again later.",
+    "api-auth-permanent-error": "An error occurred while trying to authenticate your request. Please contact your administrator.",
     "general-error": "Something went wrong. Please contact your administrator."
```

Comment:
What about `api-login-error` which is added by https://github.com/NLnetLabs/krill/pull/396 ?

Reply:
`api-auth-error` seemed to be outdated, I changed it to `api-login-error`.